### ISSUE TITLE
Certificates updated: need to change description?

### ISSUE THREAD

**Issue by ronaldtse** [riboseinc/ribose-public#9]
The certificates of the following certifications have been updated. Do we need to update anything?

* ISO 9001
* ISO 14001
* ISO/IEC 20000-1
* ISO 22301
* ISO 50001

**Comment by ribose-jeffreylau** [comment]
Yes, we do.  The "verify" links need to be updated to point to the latest reissues.
